glue_jupyter/common/toolbar_vuetify.py

```diff
@@ -0,0 +1,36 @@
+import ipyvuetify as v
+from ipywidgets import Image
+
+from glue.icons import icon_path
+
+__all__ = ['BasicJupyterToolbar']
+
+ICON_WIDTH = 20
+
+
+class BasicJupyterToolbar(v.BtnToggle):
+
+    def __init__(self, viewer):
+        super().__init__(v_model='toggle_exclusive', class_='transparent')
```

Comment:
v_model can be a string, but it will be set to the value of the selected button, by default it exclusive.